source/base/Text.ooc

```diff
@@ -174,6 +174,14 @@ Text: cover {
 		this free(Owner Receiver)
 		result
 	}
+	print: func {
+		printf("%s", this _buffer _backend pointer as Char*)
```

Comment:
`printf` works but Android defaults `stdout` to `/dev/null`. I think you can change that in a settings file somewhere, but currently we use the registered callback in `Debug print` which uses Android's logging functions.